magma/uniquification.py

```diff
@@ -56,27 +56,10 @@ def __call__(self, definition):
         else:
             if self.mode is not UniquificationMode.UNIQUIFY:
                 assert self.seen[name][key][0].name == definition.name
-            else:
-                type(definition).rename(definition, self.seen[name][key][0].name)
             self.seen[name][key].append(definition)
 
-    def _run(self, definition):
```

Comment:
This was also unnec. code, so removed. Not really related to the mentioned issue.